modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/parser/java/JavaOperationModelParser.java

```diff
@@ -406,6 +408,11 @@ public Optional<SinceMuleVersionModelProperty> getSinceMuleVersionModelProperty(
     return JavaExtensionModelParserUtils.getSinceMuleVersionModelProperty(operationElement);
   }
 
+  @Override
+  public Optional<String> getMinMuleVersion() {
+    return of(calculateFromMethod(operationElement));
```

Comment:
Is checking the method enough here? Maybe there is something at class level / field level that needs to be checked?